### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by tdrvlad** [lisphilar/covid19-sir#591] [Show and tell]
I worked on a project on Covid using your repository. I am sharing the project documentation, in case it's of use to you - it contains some images I've made in order to represent the different models.

Very good work, btw! 

![sewirf](https://user-images.githubusercontent.com/45488144/107159868-6200d080-699b-11eb-9114-12b6b78d115a.png)

[https://github.com/tdrvlad/covid_sir_modelling](url)

**Comment by lisphilar** [comment]
Dear @tdrvlad,
Thank you very much for your work and using CovsirPhy!
I read your "ModellingCovid19.pdf" and I'd like to invite you to our project as a member.
https://github.com/tdrvlad/covid_sir_modelling/blob/main/ModellingCovid19.pdf

Could your edit "example/*.ipynb" files (not the files in docs directory) to improve our documentation with the images and expalanations?
They will be automatically deployed to https://lisphilar.github.io/covid19-sir/usage_quickest.html

One comment regarding SIR-F model for more improvement:
S* (confirmed and un-categorized) is an auxiliary variable/status, not an actual. This was created for ODE modelling because we do not know the ratio of the cases who died before confirmation.
In the images for understanding, "S -> I" with `(1-alpha1) beta I` and "S-> F" with `(alpha1) beta I` could be more descriptive than "S -> S* -> I" and "S -> S* -> F" if this does not make the figure complexed.

> may travel, meet with their peers, thus easily spreading the virus.

This appears the description of "Waiting".

Thank you!

**Comment by tdrvlad** [comment]
Hello,

Thanks for your quick reply. I am glad for your proposal to contribute to
your repository. I can modify any of the models diagrams -  I agree on the
discussion on S*, in the formulas it's not an explicit state but at the
time I considered it useful in the diagram. But I can change it as you said.

So you would like me to add the images in the /example/*.ipynb files
wherever the models are mentioned? I am a little caught up with work right
now, but I can do that next week. If you want to add them, it's fine by me
also. I just want them to be of use :)

Cheers,
Vlad

**Reply by lisphilar** [reply]
Hello,
I used S* for explanation of SIR-F model for about one year, but I sometimes receive questions regarding S* meaning. It will be better for users to remove it.

Your contribution will be welcomed any time.
The images (and explanation of reproduction number Rt) will be added to "example/usage_theoretical.ipynb" file. Because SIR-F model is the main model at this time, the image for SIR-F model may be added to some other files. We will discuss this later.

Additionally, if you have a time later, please edit the other files of "example/*.ipynb" to document the analysis methods, including "Since we expect the system non-linear,...which would enable us to deploy the proposed models." and the explanation regarding RMSLE score.

Best Regards!

**Comment by tdrvlad** [comment]
Ok, I'll do this. I used to teach at a laboratory at my university so
explanations are something I enjoy doing :)

So how do we proceed, you add me as contributor and I make the changes and
push them?

Thanks!
Vlad

**Comment by lisphilar** [comment]
Thank you, I'm looking forward to collaborate with you ;)

I will add you as a collaborator and you can select the way from
- forking this repository and creating issue/pull request, and
- creating a branch with a issue and a merge request to the default branch.

The first one is explained in https://lisphilar.github.io/covid19-sir/CONTRIBUTING.html so that any users can join our project using issues/pull requests without my confirmation as the author.

**Comment by tdrvlad** [comment]
Thank you for your patience!

All the best,

**Comment by lisphilar** [comment]
FYI.
We continue to use SEWIR-F model for simulaion with set parameter values, but parameter estimation with SEWIR-F model may be deprecated because we do not have datasets regarding "Exposed" and "Waiting" as discussed in #573 and #379.

Dear @tdrvlad , how did you use this model in your analysis?

**Comment by lisphilar** [comment]
Dear @tdrvlad,
Could we use the figures in CovsirPhy documentation?

**Comment by tdrvlad** [comment]
Yes, that was the idea all along!

**Reply by lisphilar** [reply]
Dear @tdrvlad,
We discussed using the images in the documentation in the previous comments, but could you try to add a new method of `covsirphy.SIRF` which shows the image? i.e. Images will be saved in the "data" directory of this repository under Apache-2.0 license and the new method will retrieve it with `requests` pacage etc.

**Comment by tdrvlad** [comment]
Good day, sir,

Unfortunately, I am caught up with work this period. I can specifically
modify the images if that helps you but I cannot promise to modify much in
the documentation / repo. I am sorry. Besides that, you can use any of my
resources, including the text from pdf.

All the best,
Vlad
